docs/quick-start.md

```diff
@@ -40,7 +40,7 @@ RDDs have _[actions](programming-guide.html#actions)_, which return values, and
 
 {% highlight scala %}
 scala> textFile.count() // Number of items in this RDD
-res0: Long = 126
+res0: Long = 99
```

Comment:
(15 is the result you currently get when running this right?) Yes the comment is good, and you're welcome to update the results to match what you get in master now. If that's all in order I'll merge